src/main/plugins/org.dita.base/xsl/common/functions.xsl

```diff
@@ -60,6 +63,13 @@ See the accompanying LICENSE file for applicable license.
            else $doc
     "/>
   </xsl:function>
+  
+  <!-- $href (might be @href or @copy-to) has absolute or external value that should not be changed -->
+  <xsl:function name="dita-ot:unmodifiable-uri" as="xs:boolean">
```

Comment:
This is already part of another PR. The other one needs to be merged first and then this be rebased before merge.

Reply:
Yeah, I was fixing both issues in the same file on the same day, I knew some rebasing would be needed if the other PR had changes.